### PR TITLE
Update intro and goals to readme.md

### PR DESCRIPTION
**Pull request by cityhubla**
Added draft for intro for readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # project-sandbox
-a playground for experimentation
+* Project Sandbox will explore mapping the uses of publicly-owned vacant properties owned by the City of Los Angeles.
+* Project Sandbox is an example of using github resources to develop, document, track and manage and deploy civic projects with involved participation and collaboration from the general public from the beginning.
```
Comment (machikoyasuda) [inline README.md]: How about:

Project Sandbox is an example of using [GitHub](http://github.com) as an open tool to share code and data with the public, manage tasks and deploy civic projects.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+* MaptimeLA and the city will hold a series of workshops to teach and bring the project together.
```
Comment (machikoyasuda) [inline README.md]: MaptimeLA and the city will hold a series of workshops to teach the public how to use government data to create maps and answer questions.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # project-sandbox
-a playground for experimentation
+* Project Sandbox will explore mapping the uses of publicly-owned vacant properties owned by the City of Los Angeles.
```
Comment (matikin9) [inline README.md]: "Project Sandbox's goal is to map and explore the uses of publicly-owned vacant properties owned by the City of Los Angeles"?

**Comment by cityhubla** [comment]
we are going to authorize the pull and update the readme in #5

**Pull request merged**